tweb/src/main/java/io/fundreqest/platform/tweb/request/RequestController.java

```diff
@@ -200,9 +207,11 @@ public ModelAndView userRequests(Principal principal) {
         final List<RequestView> requests = mappers.mapList(RequestDto.class, RequestView.class, requestService.findRequestsForUser(principal));
         final List<PendingFundDto> pendingFunds = pendingFundService.findByUser(principal);
         final List<FaqItemDto> faqs = faqService.getFAQsForPage(FAQ_REQUESTS_PAGE);
+		boolean isAuthenticated = securityContextService.isUserFullyAuthenticated();
         return modelAndView()
                 .withObject("requests", getAsJson(requests))
                 .withObject("pendingFunds", getAsJson(pendingFunds))
+                .withObject("isAuthenticated", getAsJson(isAuthenticated))
```

Comment:
A test should be added for this (you can add it to RequestControllerTest.requests())

Reply:
Will do